### DISCUSSION TITLE
cmdline.txt options under qemu

### DISCUSSION THREAD

**Discussion by petemoore** [rsta2/circle#502] [Q&A] [answered]
Is there a way to specify cmdline.txt options when running under qemu? I'd like to specify `logdev` and `loglevel`. Or would they need to be hardcoded in the kernel under qemu? Thanks!

**Comment by rsta2** [comment, marked as answer]
Yes, there is a QEMU command line option `-append _option_`, which appends \_option\_ to the command line. Unfortunately this works only together with the `-kernel` option, so only with *qemu-system-aarch64*.

**Reply by petemoore** [reply]
Ah perfect, many thanks Rene!
